rskj-core/src/test/java/org/ethereum/util/RLPTestUtil.java

```diff
@@ -0,0 +1,70 @@
+package org.ethereum.util;
+
+import java.math.BigInteger;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.bouncycastle.util.BigIntegers.asUnsignedByteArray;
+
+public class RLPTestUtil {
+
+    public static byte[] encode(Object input) {
+        Value val = new Value(input);
+        if (val.isList()) {
+            List<byte[]> l = new ArrayList<>();
+            for (Object o:val.asList()) l.add(encode(o));
+            return RLP.encodeList(l.toArray(new byte[][]{}));
+        } else {
+            return RLP.encodeElement(toBytes(input));
+        }
+    }
+
+    /*
+     *  Utility function to convert Objects into byte arrays
+     */
+    private static byte[] toBytes(Object input) {
+        if (input == null) return null;
+        if (input instanceof byte[]) {
+            return (byte[]) input;
+        } else if (input instanceof String) {
+            String inputString = (String) input;
+            return inputString.getBytes(StandardCharsets.UTF_8);
+        } else if (input instanceof Long) {
+            Long inputLong = (Long) input;
+            return (inputLong == 0) ? ByteUtil.EMPTY_BYTE_ARRAY : asUnsignedByteArray(BigInteger.valueOf(inputLong));
+        } else if (input instanceof Integer) {
+            Integer inputInt = (Integer) input;
+            return (inputInt == 0) ? ByteUtil.EMPTY_BYTE_ARRAY : asUnsignedByteArray(BigInteger.valueOf(inputInt));
+        } else if (input instanceof BigInteger) {
+            BigInteger inputBigInt = (BigInteger) input;
+            return (inputBigInt.equals(BigInteger.ZERO)) ? ByteUtil.EMPTY_BYTE_ARRAY : asUnsignedByteArray(inputBigInt);
+        } else if (input instanceof Value) {
+            Value val = (Value) input;
+            return toBytes(val.asObj());
+        }
+        throw new RuntimeException("Unsupported type: Only accepting String, Integer and BigInteger for now");
+    }
+
+    private static final int OFFSET_SHORT_ITEM = 0x80;
```

Comment:
Just a NIT, I suggest you move these constants declaration to the beginning of the class.